driver/command/command.go

```diff
@@ -101,5 +119,49 @@ func (d *Driver) exec(op *driver.Operation) (driver.OperationResult, error) {
 		return driver.OperationResult{}, fmt.Errorf("Start of driver (%s) failed: %v", d.Name, err)
 	}
 
-	return driver.OperationResult{}, cmd.Wait()
+	if err = cmd.Wait(); err != nil {
+		return driver.OperationResult{}, fmt.Errorf("Command driver (%s) failed executing bundle: %v", d.Name, err)
+	}
+
+	result, err := d.getOperationResult(op)
+	if err != nil {
+		return driver.OperationResult{}, fmt.Errorf("Command driver (%s) failed getting operation result: %v", d.Name, err)
+	}
+
+	return result, nil
+}
+func (d *Driver) getOperationResult(op *driver.Operation) (driver.OperationResult, error) {
+	opResult := driver.OperationResult{
+		Outputs: map[string]string{},
+	}
+
+	if len(op.Outputs) == 0 {
+		return opResult, nil
+	}
+
+	for _, item := range op.Outputs {
+		fileName := path.Join(d.outputDirName, item)
+		_, err := os.Stat(fileName)
+		if err != nil {
+			if os.IsNotExist(err) {
```

Comment:
Can you add a test case that exercises this path? When an output file wasn't created?